src/components/ChatList/elements/ChannelImage.tsx

```diff
@@ -40,7 +39,7 @@ const Big: React.FC<ChannelImageMainProps> = ({type, image, style}) => {
     );
   }
 
-  if (type === 'COMMUNITY') {
+  if (type === 'TOPIC') {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The comparison value for type has been changed from 'COMMUNITY' to 'TOPIC'. Ensure that this change aligns with the intended logic and doesn't break any existing functionality.
```diff
- if (type === 'COMMUNITY') {
+ if (type === 'TOPIC') {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->